docs/en/latest/terminology/route.md

```diff
@@ -36,6 +36,12 @@ A Route mainly consists of three parts:
 2. Plugin configuration (current-limit, rate-limit)
 3. Upstream information
 
+:::note
+
+APISIX also supports GraphQL based route filtering, [see here](../../latest/router-radixtree.md#how-to-filter-route-by-graphql-attributes).
```

Comment:
While the [radix tree doc](https://apisix.apache.org/docs/apisix/router-radixtree) is indeed hard to find, I found it a bit strange to drop an admonition here just for "graphql based route filtering". In fact all these are not mentioned:

![image](https://github.com/apache/apisix/assets/39619599/36c8cdaa-3015-470a-a0fb-4a977c5db0af)

and what's missing from the above doc is the mention of APISIX variables (in addition to NGINX variables)

I would also see if we can mention these information in this doc without admonitions.

Reply:
@shreemaan-abhishek please check this

Reply:
@kayx23 instead, I just moved the whole information to the `routes` page.

Reply:
@shreemaan-abhishek ok but I wouldn't remove `router-radixtree` doc entirely - it has important information about prefix matching and so on.

![image](https://github.com/apache/apisix/assets/39619599/1d8f5b10-0b7b-4839-abcc-c3b63e384383)